Cargo.toml

```diff
@@ -16,12 +16,13 @@ exclude = ["/.github/*", "*.ipynb", "./docs/*"]
 [dependencies]
 num-traits = "0.2"
 pyo3 = { version = "0.16", optional = true }
-ndarray = { version = "0.15", optional = true }
+ndarray = "0.15"
+numpy = "0.16"
 
```

Comment:
Can we keep ndarray optional? Building just the Rust package has an extremely small dependency tree.

We just need to include `ndarray` and `numpy` in the `python` feature.

Reply:
Good catch! Done.